### ISSUE TITLE
Organize imports - improve sort order

### ISSUE THREAD

**Issue by gene-pavlovsky** [fdorg/flashdevelop#882]
Right now Code refactor "Organize imports" feature has a weird sort order.
```
import com.razorscript.math.ArithmeticError;
import com.razorscript.math.expression.supportClasses.MathFunBase;
import com.razorscript.math.MathContext;
import com.razorscript.math.RoundingMode;
```

I think it would make more sense for this to be
```
import com.razorscript.math.ArithmeticError;
import com.razorscript.math.MathContext;
import com.razorscript.math.RoundingMode;
import com.razorscript.math.expression.supportClasses.MathFunBase;
```

The current sort order especially doesn't work well with Code refactor "Separate packages" option.

**Comment by elsassph** [comment]
Well it's not weird, it's purely alphabetical, which is apparently not a good idea :)

**Comment by gene-pavlovsky** [comment]
Right :)

**Comment by gene-pavlovsky** [comment]
Closed by PR #1028
